jwala-integration-test/src/test/resources/postman/linux/httpdconf.tpl

```diff
@@ -272,8 +272,8 @@ SSLCipherSuite HIGH:MEDIUM:!aNULL:+SHA1:+MD5:+HIGH:+MEDIUM
 
 SSLSessionCacheTimeout 300
 
-SSLCertificateFile ${webServer.apacheHttpdMedia.remoteDir}/postman/ctp/app/data/security/id/${webServer.host.tokenize('.')[0].toLowerCase()}.cer
-SSLCertificateKeyFile ${webServer.apacheHttpdMedia.remoteDir}/postman/ctp/app/data/security/id/${webServer.host.tokenize('.')[0].toLowerCase()}.key
+SSLCertificateFile ${webServer.apacheHttpdMedia.remoteDir}/postman-certs/ctp/app/data/security/id/${webServer.host.tokenize('.')[0].toLowerCase()}.cer
+SSLCertificateKeyFile ${webServer.apacheHttpdMedia.remoteDir}/postman-certs/ctp/app/data/security/id/${webServer.host.tokenize('.')[0].toLowerCase()}.key
```

Comment:
change this to be same as our current path
so that we can run postman tests on any servers, we will need to run ca-cert cookbook only on 49 & 68 which has keys with passphrase